_posts/en/newsletters/2022-04-27-newsletter.md

```diff
@@ -0,0 +1,240 @@
+    been clear that CTV is really a good option".
+
+  - Russell O'Connor [noted][oconnor wizards] on #bitcoin-wizards IRC
+    that one of the proposed ways of using CTV could not be communicated
+    in an existing Bitcoin address format such as base58check,
+    [bech32][topic bech32], or bech32m.  That method of using CTV via
+    *bare script* (a script that appears directly in a scriptPubKey)
+    would require wallet developers only use bare CTV with their own
+    internal transactions or write special tools to communicate the
+    information usually contained in an address.  Alternatively, wallets
+    wanting to use CTV for some applications (such as [vaults][topic
+    vaults]) could receive payments to a P2TR address which committed to
+    later using bare CTV.
+
+    O'Connor's discussion about the address limitation was
+    [mentioned][towns bare] on list by Towns.  O'Connor
+    [replied][oconnor bare] with additional details and also noted that,
+    if support for bare CTV wasn't part of the BIP119 specification of
+    CTV, he'd advocate for a different design that would be simpler and
+    more composable (easier to combine with other opcodes to produce
+    useful scripts)---although, beyond that, he'd prefer the more
+    general `OP_TXHASH` design (see [Newsletter #185][news185 txhash]).
+    Rubin [replied][rubin bare] with several counterpoints.
+
+  - David Harding [relayed][harding transitory] a concern that CTV might
+    not receive much long-term use, either because uses for it wouldn't
+    manifest or because other covenant constructions would better serve
+    popular uses.  This would leave future consensus code developers
+    with the perpetual burden of maintaining CTV code and analyzing its
+    potential interactions with later proposed consensus changes.
+    Harding suggested temporarily adding CTV to Bitcoin for five years,
+    gathering data during that time about how people used it, and then
+    automatically disabling it unless Bitcoin users five years from now
+    decided it was worth keeping.  No respondents were in favor of this
+    approach, with most of them arguing either that the costs of this
+    approach were too high or the benefits too low.
+
+  - Antoine "darosior" Poinsot [requested][darosior apo] activating a
+    slightly modified version of [BIP118][] `SIGHASH_ANYPREVOUT`
+    ([APO][topic sighash_anyprevout]) instead of CTV, or at least prior
+    to CTV activation.  The modification to APO would allow it to
+    emulate the capabilities of CTV, although at a higher cost for some
+    applications.  Activating APO would also make it available for its
+    originally envisioned use of enabling the proposed [Eltoo][topic
+    eltoo] layer for LN, allowing for more efficient and arguably safer
+    LN channel state updates.
+
+  - James O'Beirne [suggested][obeirne benchmark] that his
+    [simple vault][] design could be used as a benchmark for evaluating
```

Comment:
Cool news, either for here or maybe next issue, that @darosior has adapted this code to work with APO: https://twitter.com/darosior/status/1518961471702642689